.github/workflows/titles.yml

```
@@ -2,7 +2,10 @@
 name: Validate Titles
 
 on:
+  # zizmor: ignore[dangerous-triggers] Limits to maintained main branch, and avoids running PR code
   pull_request_target:
+    branches:
+      - main
```

```suggestion
    branches:
      - main
      - releasebranch_*
```
@wenzeslaus indeed, it got lost somewhere, it should be like this, like the change in the labeler file.

---

In fact no, it was correct. We don't want to make it run on backported branches I think (you decide).

The comment is correct, pull_request_target is not the same as pull_request, and we are not running code from the PR, the code comes from the target branch, chosen to be "main" only, which we assume is maintained, and we would see if the file downloaded would have been vandalized and PRs submitted exploiting this.